artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/TransportConfiguration.java

```diff
@@ -328,18 +328,18 @@ private void encodeMap(final ActiveMQBuffer buffer, final Map<String, Object> ma
 
          Object val = entry.getValue();
 
-         if (val instanceof Boolean) {
+         if (val instanceof Boolean boolean1) {
```

Comment:
booleanVal / longVal / etc style naming or similar , or just leaving the casts, would be more nicer to read later than than boolean1 etc

Reply:
All should be fixed now.